Source/PlasticSourceControl/Private/PlasticSourceControlState.cpp

```diff
@@ -122,8 +122,12 @@ FName FPlasticSourceControlState::GetIconName() const
 	case EWorkspaceState::LockedByOther:
 		return FName("Perforce.CheckedOutByOtherUser");
 	case EWorkspaceState::Private: // Not controlled
-	case EWorkspaceState::Changed: // Changed but unchecked-out file is in a certain way not controlled - TODO: would need a dedicated icon
 		return FName("Perforce.NotInDepot");
+	case EWorkspaceState::Changed: // Changed but unchecked-out file is in a certain way not controlled - TODO: would need a dedicated icon
```

Comment:
Note that these two modifications are both made under the `#if ENGINE_MAJOR_VERSION == 4` from #L91, so we are missing a similar change around line #220

Reply:
That's true, I totally left those behind. It's fixed in 44a8c42 😄